c/zosaccounts.c

```diff
@@ -211,6 +308,11 @@ int userInfoGetUserId (UserInfo *info) {
   return userId;
 }
 
+/* Copy user name from User Info structure into userNameBuffer */
+void userInfoGetUserName (UserInfo *info, char *userNameBuffer) {
+  memcpy(userNameBuffer, (const char*)&info->GIDN_U_NAME, info->GIDN_U_LEN);
```

Comment:
Necessary to have its own method?

Reply:
It's just for the sake of consistency with the existing code that provides convenient access methods e.g. `userInfoGetUserId`